pkg/storage/secret/store.go

```diff
@@ -558,8 +588,11 @@ func (s *secureStore) Delete(ctx context.Context, ns string, name string) (*secr
 		return nil, false, err
 	}
 
+	// TODO: maybe we always return `true`, when there's no error?
```

Comment:
yes plus it seems it never returns the deleted secure value, so maybe a bool and error are sufficient